docs/make_work.jl

```diff
@@ -183,127 +183,32 @@ const aa = sanitize(bla, "AbstractAlgebra")
 
 bib = CitationBibliography(joinpath(@__DIR__, "oscar_references.bib"), sorting = :nyt)
 
-DocMeta.setdocmeta!(Oscar, :DocTestSetup, :(using Oscar); recursive = true)
-DocMeta.setdocmeta!(Hecke, :DocTestSetup, :(using Hecke); recursive = true)
+function doit(strict::Bool = true, local_build::Bool = false)
 
-makedocs(bib,
+  s = prod(eachline(joinpath(Oscar.oscardir, "docs", "doc.main")))
+  s = replace(s, r"\$\(aa\)" => "$aa")
+  s = replace(s, r"\$\(nemo\)" => "$nemo")
+  s = replace(s, r"\$\(hecke\)" => "$hecke")
+
+
+  doc = Meta.eval(Meta.parse(s))
```

Comment:
Is there a reason why we can't simply do
```suggestion
  doc = include(joinpath(Oscar.oscardir, "docs", "doc.main")))
```
possibly after editing `doc.main` a bit (and then renaming it, e.g. to `doc_main.jl`)